package.json

```diff
@@ -81,5 +55,34 @@
     "rimraf": "^2.5.2",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0"
-  }
+  },
+  "greenkeeper": {
+    "ignore": ["react", "react-dom"]
+  },
+  "homepage": "https://github.com/Travix-International/frint#readme",
+  "license": "MIT",
+  "main": "./lib/index.js",
+  "name": "frint",
```

Comment:
a `package.json` file not starting with the `name` key looks really weird. is it alright to have an exception for not ordering the keys alphabetically in `package.json`?